### ISSUE TITLE
What happened to this project?

### ISSUE THREAD

**Issue by tiagocpeixoto** [Azure-OSS/azure-storage-php#5]
### Summary

I was using this package in production normally, until a recent update started showing several code errors and I was no longer able to run my software. What happened to the project?

**Comment by pimjansen** [comment]
> ### Summary
> 
> 
> 
> I was using this package in production normally, until a recent update started showing several code errors and I was no longer able to run my software. What happened to the project?
> 
> 
> 
> 

We are building the project. No idea how you used it in production but it was never released as stable?

**Comment by tiagocpeixoto** [comment]
> > ### Summary
> > I was using this package in production normally, until a recent update started showing several code errors and I was no longer able to run my software. What happened to the project?
> 
> We are building the project. No idea how you used it in production but it was never released as stable?
> 
> What did you use exactly? We are planning to release it soon with an integrated flysystem adapter doe the blob storage

I´m using it in a SaaS MVP to store files (blob). I started to use it when Microsoft [archived the original repo in May](https://github.com/Azure/azure-storage-php), the occasion when I discovered this fork. 

I know there wasn't a stable version, but it worked fine for my purposes. But in the last few days the project code suddenly changed a lot and broke my code.

Thank you for your attention and for dedicating your time to this important project!

**Comment by pimjansen** [comment]
@tiagocpeixoto well first of all glad to hear you are using it.

We noticed that the original MS library was pretty bad so we decided to start over.

To continue with blobs in php we suggest to run it through flysystem though since it adds great abstraction.

As soon as this hits 1.0 there will be a flysystem adapter available. So for now the only thing i can suggest is to use the existing flysystem adapter for azure (which uses the old MS repo). Once we are ready there will be a drop'in replacement since flysystem does all the abstraction. We already tested it out and all was great.

Since you are using it in production at this moment, i would go for this route.
